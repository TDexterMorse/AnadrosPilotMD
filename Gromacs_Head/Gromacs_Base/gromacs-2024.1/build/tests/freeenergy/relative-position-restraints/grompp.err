                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/freeenergy/relative-position-restraints
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize_fft'
Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'gb-algorithm'
Ignoring obsolete mdp entry 'nstgbradii'
Ignoring obsolete mdp entry 'rgbradii'
Ignoring obsolete mdp entry 'gb-epsilon-solvent'
Ignoring obsolete mdp entry 'gb-saltconc'
Ignoring obsolete mdp entry 'gb-obc-alpha'
Ignoring obsolete mdp entry 'gb-obc-beta'
Ignoring obsolete mdp entry 'gb-obc-gamma'
Ignoring obsolete mdp entry 'gb-dielectric-offset'
Ignoring obsolete mdp entry 'sa-algorithm'
Ignoring obsolete mdp entry 'sa-surface-tension'
Ignoring obsolete mdp entry 'ns_type'
Replacing old mdp entry 'verlet-buffer-drift' by 'verlet-buffer-tolerance'
Replacing old mdp entry 'nstxtcout' by 'nstxout-compressed'
Replacing old mdp entry 'xtc-precision' by 'compressed-x-precision'

NOTE 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp]:
  Replacing vdwtype=Switch by the equivalent combination of vdwtype=Cut-off
  and vdw_modifier=Potential-switch


NOTE 2 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp]:
  With Verlet lists the optimal nstlist is >= 10, with GPUs >= 20. Note
  that with the Verlet scheme, nstlist has no effect on the accuracy of
  your simulation.


NOTE 3 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp]:
  Setting nstcalcenergy (100) equal to nstenergy (1)


WARNING 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp]:
  The switching range should be 5% or less (currently 5.56% using a
  switching range of 0.850000-0.900000) for accurate electrostatic
  energies, energy conservation will be good regardless, since ewald_rtol =
  1e-06.

Generating 1-4 interactions: fudge = 0.5

WARNING 2 [file posre.itp, line 5]:
  Some parameters for bonded interaction involving perturbed atoms are
  specified explicitly in state A, but not B - copying A to B


NOTE 4 [file topol.top, line 803]:
  You are combining position restraints with MTTK pressure coupling, which
  can lead to instabilities. If you really want to combine position
  restraints with pressure coupling, we suggest to use C-rescale pressure
  coupling instead.

Number of degrees of freedom in T-Coupling group System is 8115.00

WARNING 3 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/freeenergy/relative-position-restraints/grompp.mdp]:
  You are using pressure coupling with absolute position restraints, this
  will give artifacts. Use the refcoord_scaling option.


There were 4 NOTEs

There were 3 WARNINGs

Thanx for Using GROMACS - Have a Nice Day

