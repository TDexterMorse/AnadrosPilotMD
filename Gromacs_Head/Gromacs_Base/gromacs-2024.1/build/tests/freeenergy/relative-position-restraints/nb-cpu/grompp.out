Generated 3486 of the 3486 non-bonded parameter combinations

Generated 3486 of the 3486 1-4 parameter combinations

Excluding 3 bonded neighbours molecule type '0-LG9'

Excluding 2 bonded neighbours molecule type 'SOL'

Velocities were taken from a Maxwell distribution at 298 K
Analysing residue names:
There are:     1      Other residues
There are:   877      Water residues
Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...

The largest distance between non-perturbed excluded atoms is 0.152 nm between atom 2437 and 2438

Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 298 K

Calculated rlist for 1x1 atom pair-list as 0.900 nm, buffer size 0.000 nm

Set rlist, assuming 4x4 atom pair-list, to 0.900 nm, buffer size 0.000 nm

Note that mdrun will redetermine rlist based on the actual pair-list setup
Calculating fourier grid dimensions for X Y Z
Using a fourier grid of 36x36x36, spacing 0.091 0.091 0.091

Estimate for the relative computational load of the PME mesh part: 0.65

This run will generate roughly 0 Mb of data
